src/ngamsCore/ngamsLib/ngamsConfig.py

```diff
@@ -962,6 +962,17 @@ def getDbUseFileIgnore(self):
             return True
         return val
 
+    def getDbUsePreparedStatement(self):
+        """
+        Indicates whether to use prepared statements (default) or call SQL directly
+        from the application source code in a way that combines code and data"
+        """
+        val = self.getVal("Db[1].UsePreparedStatement")
```

Comment:
A nit-pick: could we name this `UsePreparedStatements` (plural rather than singular)? Feels more appropriate given how it applies to all queries, even the docstring above refers to them in plural.

Reply:
Done!